src/components/NewConversationDialog/NewConversationDialog.vue

```diff
@@ -328,6 +329,11 @@ export default {
 				}
 
 				await Promise.all(promises)
+
+				this.selectedParticipants.filter(participant => participant?.source === ATTENDEE.ACTOR_TYPE.EMAILS)
+					.forEach(participant => {
+						showSuccess(t('spreed', 'Invitation was sent to {actorId}', { actorId: participant.id }))
+					})
```

Comment:
It's overwhelming the screen, maybe better without it or just one general toast message ("invitations were sent" n())?

Reply:
I still believe it's better to keep just one for all numbers...

Reply:
Discarded to avoid confusions